### ISSUE TITLE
ImproperlyConfigured: Environment variable 'SUNLIGHT_API_KEY' not found.

### ISSUE THREAD

**Issue by brylie** [CiviWikiorg/CiviWiki#45]
When trying to set a development environment, I am getting the following error when running `python manage.py migrate`:

```py
django.core.exceptions.ImproperlyConfigured: Environment variable 'SUNLIGHT_API_KEY' not found.
```

The [Sunlight Labs API documentation page](https://sunlightfoundation.com/api/) indicates their services are shutting down, or have done so already.

What should I do to resolve this issue?

**Comment by CiviWiki** [comment]
This is the sunlight foundation API Key: **REDACTED**

But, since we got it, the sunlight foundation API's have been taken over by other non-profits. These are links to the two orgs OpenStates API’s (http://docs.openstates.org/en/latest/api/) and Pro-Publica Congress API (https://www.propublica.org/datastore/api/propublica-congress-api). 

You might want to message joohee on slack to get an update on where we are with the current API key because it still appears to pull down the representative data we need, but we likely need to revamp the system to use these new API's since they are being actively maintained.

**Comment by CiviWiki** [comment]
FYI, if we need to set up an API, you can set one up and share the details, or we can set one up. If it happens to need an email, it is probably best for us to use dev@civiwiki.org. Joohee currently has access to the account but I can share log in info with you too if it will streamline development.

**Comment by brylie** [comment]
@CiviWiki is that a private API key for the CiviWiki organization? If so, we should remove that comment from this discussion thread.

**Comment by JOO33** [comment]
Solution indicated in/solved by #42
